src/App/Utilities/AccountManagement/AccountsManager.cs

```diff
@@ -213,6 +215,8 @@ private async Task SwitchedAccountAsync()
                 }
                 await Task.Delay(50);
                 await _accountsManagerHost.UpdateThemeAsync();
+
+                _messagingService.Send("accountSwitched");
```

Comment:
There is a `AccountsManagerMessageCommands` that have these consts, could you pass it there? so we avoid hardcoding